modules/nodes-pods-autoscaling-creating-memory.adoc

```diff
@@ -71,7 +71,7 @@ metadata:
   namespace: default
 spec:
   scaleTargetRef:
-    apiVersion: v1 <3>
+    apiVersion: apps/v1 <3>
     kind: ReplicationController <4>
```

Comment:
Sorry I missed this the first time, but this should remain v1. In 4.8 a kind: Deployment is used here, but in 4.7 this is a ReplicationController.

Reply:
Changed to:
```
  scaleTargetRef:
    apiVersion: v1 <3>
    kind: ReplicationController <4>
    name: example <5>
```